### ISSUE TITLE
update to godot 3.5 stable 

### ISSUE THREAD

**Issue by Olm-e** [frankiezafe/gdosc#3]
hello, when trying to compile this against godot 3.5.1 stable I have several error is cascade, 
(some path missing for /gen, class def missing type of node, .... )
could be great to have it up to date ;) tky very much

**Comment by alexjsmac** [comment]
Yes I would I am on Windows and would like this very much, thanks!

**Comment by frankiezafe** [comment]
Hello, it took me some time to find the issue: this repo is a mirror of the one i maintain on gitlab: https://gitlab.com/frankiezafe/gdosc/.
I'll check this this week-end and push back on both platform.
All the best,
FZ

**Comment by frankiezafe** [comment]
Did you try https://gitlab.com/frankiezafe/gosc/?

**Comment by alexjsmac** [comment]
Okay no I don't think I've tried that one yet. I will though and let you know how it goes, thanks!

**Comment by alexjsmac** [comment]
Okay so I just compiled and tested on Linux and got the example project working with Godot 3.5.1 and PD 0.53.0!
